com.onesignal.unity.ios/Editor/BuildPostProcessor.cs

```diff
@@ -152,6 +160,8 @@ public class BuildPostProcessor : IPostprocessBuildWithReport {
             var targetGuid = _project.GetMainTargetGuid();
             var targetName = _project.GetMainTargetName();
 
+            _project.SetBuildProperty(targetGuid, "ENABLE_BITCODE", "NO");
```

Comment:
This method is named `AddProjectCapabilities` which shouldn't touch something unrelated like bitcode. I recommend moving this to the new `DisableBitcode()` method I suggested above.